### PR TITLE
Update LXD steps, add 2.4.0 version

### PR DESCRIPTION
**Pull request by battlemidget**
cc @johnsca 

@evilnick Can we also get the 2.4.0 branch built from master and published as well?

Signed-off-by: Adam Stokes <battlemidget@users.noreply.github.com>

### CODE REVIEW

File: en/index.md [hunk 1]
```diff
@@ -35,24 +35,23 @@ For **localhost** deployments the following setup is recommended:
...
+- 250G SSD
 
 ## Installing conjure-up
 
 `conjure-up` is available on Ubuntu Xenial 16.04 LTS and macOS
 
 ### Ubuntu
 ```bash
-sudo snap install conjure-up --classic
+$ sudo snap install conjure-up --classic
```
Comment (degville) [inline en/index.md]: The docs team has standardised on omitting the prompt and relying on design if that's a requirement.

File: en/index.md [hunk 2]
```diff
@@ -74,26 +73,115 @@ If you have **conjure-up** already installed, you can update to a different
...
+$ sudo snap install lxd
+/snap/bin/lxd init --auto  #  Notice there is NO __sudo__ prepended here.
+```
+
+### LXD Storage
+
+At least one storage pool must be created and the default profile be set to use that storage device:
+
+```
+$ /snap/bin/lxc storage list
```
Comment (degville) [inline en/index.md]: We also split the input/output (to help with copy/paste, etc.).

File: en/index.md [hunk 3]
```diff
@@ -74,26 +73,115 @@ If you have **conjure-up** already installed, you can update to a different
...
-sudo snap refresh conjure-up --classic --beta
+$ sudo snap refresh conjure-up --classic --beta
 ```
 
+## Users of LXD
+
+**conjure-up** requires that the minimum version of LXD be **2.18**. Additionally,
+LXD should be configured prior to running.
+
+### Recommended LXD Setup
```
Comment (degville) [inline en/index.md]: I'd change this to `### Install LXD`

**Comment by battlemidget** [comment]
@degville let me know if that's what you meant by spliiting command and output

**Comment by evilnick** [comment]
Yup, that looks better - the command and output will render as separate blocks

**Comment by evilnick** [comment]
I'll merge this and add a 2.4.0 branch then - I guess we want to push these changes to devel also?

**Comment by battlemidget** [comment]
> I'll merge this and add a 2.4.0 branch then - I guess we want to push these changes to devel also?

Yes please to devel

**Pull request merged**